pykokkos/interface/views.py

```diff
@@ -503,8 +516,10 @@ def from_numpy(array: np.ndarray, space: Optional[MemorySpace] = None, layout: O
     else:
         ret_list = list((array.shape))
 
-
-    return View(ret_list, dtype, space=space, trait=Trait.Unmanaged, array=array, layout=layout)
+    # Temporary Fix. Will revert when fixed
+    view = pk.View(ret_list, dtype)
+    view[:] = array
+    return view
```

Comment:
As discussed, create a custom `asarray` in GuassianNB, and use it where necessary